dockstore-webservice/src/main/java/io/dockstore/webservice/languages/ServicePrototypePlugin.java

```diff
@@ -63,6 +64,19 @@ public VersionTypeValidation validateWorkflowSet(String initialPath, String cont
                 if (files == null) {
                     validationMessageObject.put(initialPath, "The key 'files' does not exist.");
                     isValid = false;
+                } else {
+                    // check that files in .dockstore.yml exist
+                    ArrayList<String> missingFiles = new ArrayList<String>();
```

Comment:
Use List on the left unless you really rely on ArrayList
https://java2blog.com/java-list-vs-arraylist/